### PR TITLE
feat: add Bubs Testnet

### PR DESCRIPTION
**Pull request by jcstein**
This is for a PR to add [Bubs Testnet](https://bubstestnet.com) to the chain registry.

### CODE REVIEW

File: _data/chains/eip155-1582.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        "https://tarotestnet.com"
+    ],
+    "nativeCurrency": {
+      "name": "Ether",
+      "symbol": "ETH",
+      "decimals": 18
+    },
+    "infoURL": "https://tarotestnet.com",
+    "shortName": "eth",
+    "chainId": 1,
```
Comment (jcstein) [inline _data/chains/eip155-1582.json:17]:
```suggestion
    "chainId": 1582,
```

**Review by alokeutpal — APPROVED** [review]
Y

**Review by alokeutpal — APPROVED** [review]

**Comment by jcstein** [comment]
yooo

**Comment by jcstein** [comment]
👀 @ligi 🙏

**Comment by ligi** [comment]
failing CI -> no review

**Comment by jcstein** [comment]
scrot 2023-10-25 at 3 10 01 PM
how to run CI?

File: _data/chains/eip155-1582.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "networkId": 1,
```
[the remark this answers is not in the document]
Comment (nnnnnnsssss) [reply _data/chains/eip155-1582.json]: Recheck[]()

**Pull request closed without merging**